4_gap_analysis/in/Agency.csv

```diff
@@ -0,0 +1,146 @@
+Full,SIR ,WQX,State,Type
```

Comment:
Consider renaming `SIR` to something a little more detailed for posterity. for example, `ID_from_SIR` or `Agency_Abbv`